src/main/java/hwr/oop/group4/todo/ui/ProjectUi.java

```diff
@@ -0,0 +1,211 @@
+package hwr.oop.group4.todo.ui;
```

Comment:
We need to do something about the length of this class

Reply:
See https://github.com/yxyx-github/hwr-oop-group-4-todo/pull/32#discussion_r1183548172